### PR TITLE
Add mdaencore to osx_arm64.txt migrations list

### PR DESCRIPTION
**Pull request by IAlibay**
For the following feedstock, thanks :)

https://github.com/conda-forge/mdaencore-feedstock

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@ocefpaf  is it possible to have the pre-commit.ci just sort things for the contributors? It feels a little silly to ask them to do it.

File: recipe/migrations/osx_arm64.txt [hunk 1]
```diff
@@ -607,6 +607,7 @@ mcpl
 mcstas-suite
 mdanalysis
 mdanalysistests
+mdaencore
```
Comment (hmaarrfk) [inline recipe/migrations/osx_arm64.txt:610]:
```suggestion
mdaencore
mdanalysis
mdanalysistests
```

**Comment by xhochy** [comment]
> @ocefpaf  is it possible to have the pre-commit.ci just sort things for the contributors? It feels a little silly to ask them to do it.

You can command it with "pre-commit.ci autofix".

**Comment by hmaarrfk** [comment]
Where do I see the precommit-ci configuration

**Pull request merged**